discovery-provider/src/queries/get_health.py

```diff
@@ -702,3 +702,52 @@ def get_latest_chain_block_set_if_nx(redis=None, web3=None):
             )
 
     return latest_block_num, latest_block_hash
+
+
+def get_acdc_status() -> Tuple[Dict, bool]:
+    # setup return objects
+    res = {
+        "current_block": None,
+        "current_block_number": None,
+        "signers": None,
+        "signers_count": None,
+        "snapshot": None,
+        "me": None,
+        "is_signer": None,
+        "health": None,
+    }
+
+    # gather connections
+    web3endpoint = os.getenv("audius_web3_nethermind_rpc")
+    web3 = get_nethermind_web3(web3endpoint)
+
+    # web3 get current block
+    current_block = web3.eth.get_block('latest')
+    res["current_block"] = current_block
+    res["current_block_number"] = current_block.number
+
+    # clique get signers
+    signers = rpc("clique_getSigners")
+    res["signers"] = signers
+    res["signers_count"] = len(signers)
+
+    # clique get snapshot
+    res["snapshot"] = rpc("clique_getSnapshot")
+
+    # net_getLocal
+    me = rpc("net_getLocal")
```

Comment:
what is me and `net_getLocal`?

Reply:
gets the address of the current node so we can check if it's a signer

Reply:
i tried calling and it didn't recognize, is there documentation?